static/app/components/profiling/exportProfileButton.tsx

```diff
@@ -10,12 +10,12 @@ import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 
 interface ExportProfileButtonProps
-  extends Omit<LinkButtonProps, 'onClick' | 'children' | 'external'> {
+  extends Omit<LinkButtonProps, 'title' | 'onClick' | 'children' | 'external'> {
   eventId: string | undefined;
   orgId: string;
   projectId: string | undefined;
   children?: React.ReactNode;
-  variant?: 'xs' | 'default';
+  variant?: 'zero';
```

Comment:
we still allow the `size` prop, which does get passed as `md` somewhere, but we also have a `variant` that is also a size? But when `variant` is `zero`, we render a `DownloadButton`, but if not we render a LinkButton with size zero 🤯 . I’m so confused now 😅

Reply:
I think we can drop this entirely. I have probably written this code back when we were merging profiling into Sentry and I was struggling to grok the codebase in the first place